### DISCUSSION TITLE
Item Counts and Quick Statistics

### DISCUSSION THREAD

**Discussion by leelooauto** [sysadminsmedia/homebox#59] [Ideas]
It would be great to see a few quality of life improvements although I get what is important to me may not be important to others.

Home location overview does not currently pull through item counts from nested locations
Viewing nested locations does not show any item counts 
Quick statistics currently only shows in home, it would be great to also have this by location, specifically for me I'm looking for total value of that location including all sub locations plus a value or quick statistic at each nested location, why? because insurance.

**Comment by rossp** [comment]
This would be very helpful - and like you it's important for me for insurance purposes. This could also be achieved by enhancing the 'search' capability to show aggregate quantities or values for search results.
